k8s/src/test/scala/io/buoyant/k8s/IngressCacheTest.scala

```diff
@@ -352,12 +402,21 @@ class IngressCacheTest extends FunSuite with Awaits {
   val annotationClass = "linkerd"
 
   def mkIngressApiServiceReturning(response: String) = Service.mk[Request, Response] {
+    mkWatchedIngressApiServiceReturning(response, "")
+  }
+
+  def mkWatchedIngressApiServiceReturning(response: String, watchResponse: String) = Service.mk[Request, Response] {
     case req if req.uri.contains("/apis/extensions/v1beta1/ingresses") =>
       val rsp = Response()
       rsp.content = Buf.Utf8(response)
       Future.value(rsp)
     case req if req.uri.contains("/apis/extensions/v1beta1/watch/ingresses") =>
```

Comment:
This case block could probably use some close review. I mostly blindly copied and pasted from `EndpointsNamerTest` until it worked, so I'm sure it could be done better. 🤓